packages/backend/server/src/core/doc/storage/doc.ts

```diff
@@ -1,3 +1,4 @@
+import { Logger } from '@nestjs/common';
```

Comment:
after making merge service standalone, we no longer need the low level `DocStorageAdapter` anymore, could be removed and implemented in a simpler way

Reply:
It was designed to share the Adapter logic with `nbstore`, but looks nowadays, it brings too much maintenance cost than the benefits. We didn't make it even happens to reference `nbstore` directly yet.

Another app for example `electron`, has already giving-up implementing `nbstore` but just provide the apis it requires to avoid cyclic references.

Reply:
Keep it not change this time to avoid blocking merges and iteration releases? I will refactor `DocStorageAdapter` in the next iteration.

Reply:
cool

Reply:
trace on CLOUD-116